common/src/main/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtils.java

```diff
@@ -54,71 +72,188 @@ private static boolean isValidStartingArgument(final String argument) {
+                "-XX:-UseParallelScavenge"
+        };
     }
 
     /**
      * Returns an array containing the starts of valid (ie safe and supported)
      * arguments for the JVM
-     *
+     * <p>
      * Based on
-     * http://java.sun.com/javase/6/docs/technotes/guides/javaws/developersguide/syntax.html
+     * https://docs.oracle.com/javase/8/docs/technotes/guides/javaws/developersguide/syntax.html#secure-property
      */
     private static String[] getValidStartingVMArguments() {
         return new String[]{
-            "-ea", /* enable assertions for classes */
-            "-enableassertions", /* enable assertions for classes */
-            "-da", /* disable assertions for classes */
-            "-disableassertions", /* disable assertions for classes */
-            "-verbose", /* enable verbose output */
-            "-Xms", /* set initial Java heap size */
-            "-Xmx", /* set maximum Java heap size */
-            "-Xss", /* set java thread stack size */
-            "-XX:NewRatio", /* set Ratio of new/old gen sizes */
-            "-XX:NewSize", /* set initial size of new generation */
-            "-XX:MaxNewSize", /* set max size of new generation */
-            "-XX:PermSize", /* set initial size of permanent gen */
-            "-XX:MaxPermSize", /* set max size of permanent gen */
-            "-XX:MaxHeapFreeRatio", /* heap free percentage (default 70) */
-            "-XX:MinHeapFreeRatio", /* heap free percentage (default 40) */
-            "-XX:UseSerialGC", /* use serial garbage collection */
-            "-XX:ThreadStackSize", /* thread stack size (in KB) */
-            "-XX:MaxInlineSize", /* set max num of bytecodes to inline */
-            "-XX:ReservedCodeCacheSize", /* Reserved code cache size (bytes) */
-            "-XX:MaxDirectMemorySize",};
+                "-ea", /* enable assertions for classes */
+                "-enableassertions", /* enable assertions for classes */
+                "-da", /* disable assertions for classes */
+                "-disableassertions", /* disable assertions for classes */
+                "-verbose", /* enable verbose output */
+                "-Xms", /* set initial Java heap size */
+                "-Xmx", /* set maximum Java heap size */
+                "-Xss", /* set java thread stack size */
+                "-XX:NewRatio", /* set Ratio of new/old gen sizes */
+                "-XX:NewSize", /* set initial size of new generation */
+                "-XX:MaxNewSize", /* set max size of new generation */
+                "-XX:PermSize", /* set initial size of permanent gen */
+                "-XX:MaxPermSize", /* set max size of permanent gen */
+                "-XX:MaxHeapFreeRatio", /* heap free percentage (default 70) */
+                "-XX:MinHeapFreeRatio", /* heap free percentage (default 40) */
+                "-XX:UseSerialGC", /* use serial garbage collection */
+                "-XX:-UseSerialGC",
```

Comment:
Is this really a prefix argument or should this be moved to the valid arguments list?

Reply:
That's how it is in doc.